### ISSUE TITLE
MD5 errors with node v6.7.0

### ISSUE THREAD

**Issue by tsantero** [joyent/node-triton#138]
For users on macs who install node via homebrew, today they'll be getting the latest, which is v6.7.0

```
➜  ~ node --version
v6.7.0
```

Unfortunately, simple operations such as creating an instance fail with `BadDigestError`s e.g.:

```
{"name":"triton","hostname":"tomsantero","pid":71109,"level":10,"host":"us-west-1.api.joyent.com","msg":"close cloudapi http client","time":"2016-10-12T18:41:18.343Z","src":{"file":"/usr/local/lib/node_modules/triton/lib/cloudapi2.js","line":126,"func":"close"},"v":0}
triton instance create: error (BadDigest): BadDigestError: Content-MD5
    at IncomingMessage.finish (/usr/local/lib/node_modules/triton/lib/SaferJsonClient.js:120:26)
    at IncomingMessage.g (events.js:291:16)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

downgrading to LTS v4.6.0 ameliorates the issue, and as such is a non-blocker for me; however, I thought I'd bring this to your attention.

**Comment by doublerebel** [comment]
Looks like this is a change in Node 5/6.  Best described at [nodejs/node#8853](https://github.com/nodejs/node/issues/8853):

> The default string encoding changed in node v6.0.0 (from 'binary' to 'utf8'). So you need to explicitly specify 'binary'/'latin1' if that is what you are passing to update():
> 
> console.log(crypto.createHash('md5').update('\xBB', 'binary').digest('hex'))

**Comment by cburroughs** [comment]
This should have been resolved with 84338a44b2314225a4233dd1907bf1e4c530012a  in 4.14.2 or later.  Please try upgrading and reopen if you still see this issue.
